docs/reference/settings.md

```diff
@@ -809,6 +809,11 @@ WAGTAIL_WORKFLOW_ENABLED = True
 
 Specifies whether moderation workflows are enabled (default: `True`). When disabled, editors will no longer be given the option to submit pages to a workflow, and the settings areas for admins to configure workflows and tasks will be unavailable.
 
+```{versionchanged} 5.2
+
+This setting has been deprecated. Use `WAGTAIL_WORKFLOW_ENABLED` instead.
+```
+
```

Comment:
I gave this as an example of how we've used `versionchanged` in the past. I didn't mean to put this exact same text back :-)

Reply:
oh my bad , actualy i was a bit confused on this part , ill edit rightaway

Reply:
done 👍